api/app/reviews/tests.py

```diff
@@ -1465,27 +1439,47 @@ def static_seed_data(self):
         db.session.commit()
 
         self.first_headers = self.get_auth_header_for("firstuser@mail.com")
-        # self.other_headers = self.get_auth_header_for("someuser@mail.com")
 
         db.session.flush()
 
     def test_get_reference_request_by_response_id(self):
+        self.seed_static_data()
+        reference_req = ReferenceRequest(1, 'Mr', 'John', 'Snow', 'Supervisor', 'common@email.com')
+        reference_request_repository.create(reference_req)
+        REFERENCE_DETAIL = {
+            'token': reference_req.token,
+            'uploaded_document': 'DOCT-UPLOAD-78999',
+        }
+        # response = self.app.get(
+        #     '/api/v1/reference', data=REFERENCE_DETAIL, headers=self.first_headers)
+        # self.assertEqual(response.status_code, 201)
+
+        params = {'event_id': 1, 'application_form_id': 1, 'user_id:': 5, 'sort_column' : 'review_response_id',
+                  'reference_detail': REFERENCE_DETAIL} # confirm params, check ids
+
+        response = self.app.get('/api/v1/review', headers=self.get_auth_header_for('r1@r.com'), data=params)
 
+        data = json.loads(response.data)
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(len(data), 2) # Check length
 
+    def test_get_reference_request_with_two_references(self):
+        # Initial draft / planning

```

Comment:
This still needs a lot of work - I think I will be able to improve it once I completely understand the previous test.

Reply:
Good morning,@avishkar58! Would you mind having another look at my test, please?

I've also been wondering, how does one debug / test their test normally?